commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,22 @@
      }
 
     /**
+     * calculates the page number of the result.
+     *
+     * @return the page number of the result.
+     */
+    default long getPage() {
+
+        if (getOffset() == null) {
```

Comment:
I think I understand your intention, but I'm not sure if throwing an ÙnsupportedOperationException` is a good idea here. We already use `Long` everywhere in this class and they aren't nullable. I would suggest to use `Long` here two.